rflib/chipcon_nic.py

```diff
@@ -986,7 +986,7 @@ def calculatePktChanBW(self, mhz=24, radiocfg=None):
         #minbw = (2 * freq_uncertainty) + self.getMdmDRate() # uncertainty for both sender/receiver
         minbw = (self.getMdmDRate() + freq_uncertainty)
 
-        possibles = [ 53e3,63e3,75e3,93e3,107e3,125e3,150e3,188e3,214e3,250e3,300e3,375e3,428e3,500e3,600e3,750e3, ]
+        possibles = [63e3,75e3,93e3,107e3,125e3,150e3,188e3,214e3,250e3,300e3,375e3,428e3,500e3,600e3,750e3, ]
```

Comment:
is there something wrong with 53e3?

Reply:
I responded via email, looks like it ended up at the bottom of the comment chain. Deleted and added here for relevance.

 can answer one of these easily, there is an issue with 53e3, minimum is
54e3 per table 63 (24mhz crystal) page 192 cc1110fx/cc1111fx datasheet.

I personally have experienced issues with 53e3 so I just removed it, should
have put in 54e3. I will fix that.

Reply:
awesome.  thanks!